internal/cfg/config.go

```diff
@@ -85,6 +86,21 @@ func (gc *GlobalConfig) Merge(other *GlobalConfig) error {
 	return nil
 }
 
+func (gc *GlobalConfig) Validate() error {
```

Comment:
You could re-use `github.com/hashicorp/go-multierror`. It essentially does what you're doing here +/-. Wouldn't need to implement your own logic :smile:

Reply:
Updated